### PR TITLE
Update autobuild functionality

### PR DESCRIPTION
**Pull request by asteel-gsa**
Bug when determining main script

### CODE REVIEW

File: .github/workflows/build-image.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@
 name: Build and Publish ClamAV Image
 on:
     workflow_dispatch:
-    workflow_call:
+    schedule:
+      - cron: '0 4 * * 0'
```
Comment (sambodeme) [inline .github/workflows/build-image.yml:6]: What is the issue number related to this PR ?
Is the intent to build this image once a week ?

Comment (jadudm) [reply .github/workflows/build-image.yml:6]: I don't know what the issue number is, but I do know that it is intended to have weekly image builds.

Comment (asteel-gsa) [reply .github/workflows/build-image.yml:6]: @sambodeme https://github.com/GSA-TTS/FAC/issues/1064

Not sure what happened here: https://github.com/GSA-TTS/clamav-rest/actions/runs/5167122077 but testing this way for the schedule.

Comment (sambodeme) [reply .github/workflows/build-image.yml:6]: @asteel-gsa  looks like the job run into a permission issue read vs write (which I believe is to be addressed outside of this PR). 
Regarding the weekly goal, this cron does achieve it (every Sunday 4am).

**Review by sambodeme — APPROVED** [review]

**Pull request merged**